styleguide/gulpfile.js

```diff
@@ -66,6 +68,34 @@ const _sassCompile = () => {
   return combined;
 };
 
+/*
+* Get the spotimages map.
+*/
+const _spotimagesMap = () => {
+  // require colors
+  const colors = require('./components/11-base/colors/colors.config.js');
+  const map = [];
+
+  map.push({
+    'suffix': '--default',
+    'colors': [ Color('#009DE0') ]
+  });
+
+  if (colors) {
+    const colormap = colors.context.secondary;
+    for (const cs in colormap) {
```

Comment:
Wouldn't the ES map function be more appropriate?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map